flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointConfigInfo.java

```diff
@@ -65,6 +65,8 @@ public class CheckpointConfigInfo implements ResponseBody {
     public static final String FIELD_NAME_CHECKPOINTS_AFTER_TASKS_FINISH =
             "checkpoints_after_tasks_finish";
 
+    public static final String FIELD_NAME_STATE_CHANGELOG = "state_changelog";
```

Comment:
I think `changelog_state_enabled` looks better.

Reply:
How about 'state_changelog_enabled' ?

Reply:
Hi @Myasuka , I change it to 'state_changelog_enabled'.